dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/features/AccountApiMockTest.java

```diff
@@ -0,0 +1,67 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.dimensiondata.cloudcontrol.features;
+
+import org.jclouds.dimensiondata.cloudcontrol.domain.Account;
+import org.jclouds.dimensiondata.cloudcontrol.internal.BaseDimensionDataCloudControlMockTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.AssertJUnit.assertEquals;
+
+@Test(groups = "unit", testName = "AccountApiMockTest", singleThreaded = true)
+public class AccountApiMockTest extends BaseDimensionDataCloudControlMockTest {
+
+   @BeforeMethod
+   @Override
+   public void start() throws IOException {
+      super.start();
+   }
+
+   @Test
+   public void testGetAccount() throws Exception {
+      server.enqueue(jsonResponse("/account.json"));
+      Account account = api.getAccountApi().getMyAccount();
+      assertNotNull(account);
+
+      assertEquals("devlab1-online-vendor-devuser1", account.userName());
+      assertEquals("Donal Lunny", account.fullName());
+      assertEquals("Donal", account.firstName());
+      assertEquals("Lunny", account.lastName());
+      assertEquals("devlab1-online-vendor-devuser1@opsource.net", account.emailAddress());
+      assertThat(account.roles().contains(Account.RoleType.create("storage")));
+      assertThat(account.roles().contains(Account.RoleType.create("primary administrator")));
+      assertEquals("1", account.phone().countryCode());
+      assertEquals("1234567", account.phone().number());
+      assertEquals("Marketing", account.department());
+      assertEquals("custom defined 1", account.customDefined1());
+      assertEquals("custom defined 2", account.customDefined2());
+      assertEquals("6ac1e746-b1ea-4da5-a24e-caf1a978789d", account.organization().id());
+      assertEquals("Eng Customer of DevLab1-homed online sign-up Vendor", account.organization().name());
+      assertEquals("DevLab1Geo1", account.organization().homeGeoName());
+      assertEquals("apidevlab1.opsourcecloud.net", account.organization().homeGeoApiHost());
+      assertEquals("DEVLAB1GEO1", account.organization().homeGeoId());
+      assertEquals("NORMAL", account.state());
+
+      assertSent(server, "GET", "/caas/2.4/user/myUser");
+   }
+
```

Comment:
Add another test method that simulates a 404 response, to verify that the method returns `null`.
Since you are defining an explicit `@Fallback` in the api method, we'd better verify it behaves as expected on the affected responses.